recipe/deploy/vendors.php

```diff
@@ -23,26 +23,49 @@
 set('composer_channel', null);
 
 set('bin/composer', function () {
+    $composerVersionToInstall = get('composer_version', null);
+    $composerChannelToInstall = get('composer_channel', null);
+    $composerValidChannels = ['stable', 'snapshot', 'preview', '1', '2',];
+    if ($composerChannelToInstall && !in_array((string)$composerChannelToInstall, $composerValidChannels, true)) {
+        throw new \Exception('Selected Composer channel ' . $composerChannelToInstall . ' is not valid. Valid channels are: ' . implode(', ', $composerValidChannels));
```

Comment:
Well, after thinking, let's omit this part. Let's install stable composer and allow user yo override it is more user friendly.

Reply:
Can you plz check if the latest version is what you were thinking about ?